pyodbc.cpython-310-aarch64-linux-gnu.so,sha256=Z3Sc14B_LL_Jtr_72ZgHEDc1Mfvytcf2EntT17cRmIQ,989425
pyodbc.pyi,sha256=rx05N8EYqujKI3FWtMikUjqMPTv4VQg0ChXny9DaisE,32326
pyodbc-5.0.0.dist-info/top_level.txt,sha256=Rb69vS-SNN_Bfh99vX7eCcvEAJoyzowG3u4R7W36Flw,8
pyodbc-5.0.0.dist-info/WHEEL,sha256=mUS4hxNBfx59QyErIFB-zSc7bBe3KLfW0jXT5R7XFVw,154
pyodbc-5.0.0.dist-info/RECORD,,
pyodbc-5.0.0.dist-info/METADATA,sha256=UCCQ_QLPO9Mamlo69HQstSc1QzMfiCYTSUJZIm2clXA,2717
pyodbc-5.0.0.dist-info/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
